lib/postgres_v3_experimental.dart

```diff
@@ -40,11 +40,23 @@ abstract class PgSession {
     Duration? timeout,
   });
 
+  /// Executes the [query] with the given [parameters].
+  ///
+  /// [query] must either be a [String] or a [PgSql] query with types for
+  /// parameters. When a [PgSql] query object is used, [parameters] can be a
+  /// list of direct values. Otherwise, it must be a list of [PgTypedParameter]s.
```

Comment:
s/list/list or map/ ?

Reply:
I've added a sentence to explain that maps can be used with `PgSql.map` objects used as parameters.